src/Catalog/Dnx/DnxCatalogCollector.cs

```diff
@@ -255,6 +298,8 @@ private async Task<bool> AreRequiredPropertiesPresentAsync(Storage destinationSt
                 // If the ETag's differ, we'll fall back to using a single HTTP GET request.
                 var token1 = await _sourceStorage.GetOptimisticConcurrencyControlTokenAsync(sourceUri, cancellationToken);
 
+                telemetryProperties[TelemetryConstants.SizeInBytes] = sourceBlob.Length.ToString();
+
```

Comment:
Should `ProcessPackageDetailsViaHttpAsync` also add this property?

Reply:
Yes, somehow I removed it.  Adding it back.